lib/js_of_ocaml.ml

```diff
@@ -17,7 +17,7 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  *)
 
-module CSS = CSS
+module CSS = CSS__Internal
```

Comment:
You should not use `__` in your own module names. This is just a naming convention that's taken up by dune's wrapping mechanism. You should use some name like `css_internal` or w/e

Reply:
@hhugo is there any name for the module you’d prefer?